doc/sphinx/source/input.rst

```diff
@@ -102,7 +122,7 @@ A list of the datasets for which a cmorizers is available is provided in the fol
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
 | Eppley-VGPM-MODIS            | intpp (Omon)                                                                                         |   2  | Python          |
 +------------------------------+------------------------------------------------------------------------------------------------------+------+-----------------+
-| ERA5                         | clt, evspsbl, evspsblpot, mrro, pr, prsn, ps, psl, ptype, rls, rlds, rsds, rsdt, rss, uas, vas, tas, |   3  | Python          |
+| ERA5                         | clt, evspsbl, evspsblpot, mrro, pr, prsn, ps, psl, ptype, rls, rlds, rsds, rsdt, rss, uas, vas, tas, |   3  | ESMValTool      |
```

Comment:
If **Python** doesn't make sense for 'CMORization on the fly', we can just leave it empty. However, we provide a diagnostic i.e. era5.py which is in Python. Also, we can add * next to the dataset name e.g. ERA5* and explain in the header of this table that the dataset is supported by "CMORization on the fly".